src/components/CurrentSong.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import PropTypes from "prop-types";
+import DefaultCover from "../assets/cover_placeholder.gif";
+
+const CurrentSong = props => (
+  <div className={props.playing ? "thumb shown" : "thumb"} id="metaDisplay">
+    <img
+      alt="album art"
+      data-meta="picture"
+      src={props.fastConnection ? props.currentSong.art : DefaultCover}
+    />
+    <div id="nowPlaying">
+      <progress
+        data-meta="duration"
+        max={props.songDuration}
+        value={props.durationVal}
+      />
+      <div data-meta="title">{props.currentSong.title}</div>
+      <div data-meta="artist">{props.currentSong.artist}</div>
+      <div data-meta="album">{props.currentSong.album}</div>
+      <div data-meta="listeners">Listeners: {props.listeners}</div>
+      {props.mountOptions}
```

Comment:
I don't think that mount options should be part of this component. I know that it sits now in the same container but logically it's not related to the "current song". I would try to find a way to take out the mount options from this component.